### ISSUE TITLE
fontawesome bluesky brand icon not appearing in rmarkdown navbar

### ISSUE THREAD

**Issue by shannonpileggi** [rstudio/rmarkdown#2581]
## Checklist

When filing a _bug report_, please check the boxes below to confirm that you have provided us with the information we need. Have you:

- [x] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?

- [x] included a minimal, self-contained, and reproducible example?

- [x] pasted the output from `xfun::session_info('rmarkdown')` in your issue?

- [x] upgraded all your packages to their latest versions (including your versions of R, the RStudio IDE, and relevant R packages)?

- [x] installed and tested your bug with the development version of the rmarkdown package using `remotes::install_github("rstudio/rmarkdown")` ?

# 

I have followed the reprex from the previously reported #1991 on a similar but resolved issue.

I am trying to include a [bluesky icon](https://fontawesome.com/icons/bluesky?f=brands&s=solid) on my distill website and cannot get it to show. I am posting a reprex in rmarkdown only.
Please let me know if I have missed any updates that would resolve this issue. 

## Reprex

```
path <- tempfile()
dir.create(path)
index <- file.path(path, "index.Rmd")
file.create(index)
config <- list(
  navbar = list(
    left = list(
      list(
        text = "Bluesky",
        icon = "fa-bluesky",
        href = "index.html"
      ),
      list(
        text = "Bluesky",
        icon = "fa fa-bluesky",
        href = "index.html"
      ),
      list(
        text = "Bluesky",
        icon = "fas fa-bluesky",
        href = "index.html"
      ),
      list(
        text = "Bluesky",
        icon = "fab fa-bluesky",
        href = "index.html"
      ),
      list(
        text = "GitHub",
        icon = "fa-github",
        href = "index.html"
      ),
      list(
        text = "GitHub",
        icon = "fa fa-github",
        href = "index.html"
      ),
      list(
        text = "GitHub",
        icon = "fab fa-github",
        href = "index.html"
      )
    )
  ),
  output = "html_document"
)
site <- file.path(path, "_site.yml")
yaml::write_yaml(config, site)
html <- rmarkdown::render_site(index)
html <- file.path(path, html)
browseURL(html)
```

Result:
<img width="514" alt="image" src="https://github.com/user-attachments/assets/c2859a77-ad66-4f4b-bf9c-1694f2a42bf2">

## Session info 

I tried this on both rmarkdown 2.29 and the dev version.

```
> xfun::session_info('rmarkdown')
R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 22631), RStudio 2024.9.0.375

Locale:
  LC_COLLATE=English_United States.utf8 
  LC_CTYPE=English_United States.utf8   
  LC_MONETARY=English_United States.utf8
  LC_NUMERIC=C                          
  LC_TIME=English_United States.utf8    

Package version:
  base64enc_0.1.3   bslib_0.8.0       cachem_1.1.0     
  cli_3.6.3         digest_0.6.37     evaluate_1.0.1   
  fastmap_1.2.0     fontawesome_0.5.2 fs_1.6.5         
  glue_1.8.0        graphics_4.4.2    grDevices_4.4.2  
  highr_0.11        htmltools_0.5.8.1 jquerylib_0.1.4  
  jsonlite_1.8.9    knitr_1.49        lifecycle_1.0.4  
  memoise_2.0.1     methods_4.4.2     mime_0.12        
  R6_2.5.1          rappdirs_0.3.3    rlang_1.1.4      
  rmarkdown_2.29.1  sass_0.4.9        stats_4.4.2      
  tinytex_0.54      tools_4.4.2       utils_4.4.2      
  xfun_0.49         yaml_2.3.10      

Pandoc version: 3.2
```

Thank you very much for any assistance! <3

**Comment by cderv** [comment]
This icon is available in fontawesome 6.5.2. **rmarkdown** uses the **fontawesome** package and it seems it has only 6.4.2 for CRAN version. 

6.5.2 is available in dev version: https://github.com/rstudio/fontawesome/pull/119, assuming it works (https://github.com/rstudio/fontawesome/issues/121) 

So this is something related to this dependency, unless you are loading your fontawesome dependency differently.

**Comment by shannonpileggi** [comment]
thank you so much @cderv! i apologize for not taking the time to track down that missing link. everything worked great with the dev version of {fontawesome}. <3
